examples/linear_model/plot_ols_ridge_variance.py

```diff
@@ -60,12 +56,12 @@
     ax.plot(X_test, clf.predict(X_test), linewidth=2, color='blue')
     ax.scatter(X_train, y_train, s=30, c='red', marker='+', zorder=10)
 
-    ax.set_xticks(())
-    ax.set_yticks(())
+    ax.set_title(name)
+    ax.set_xlim(0, 2)
     ax.set_ylim((0, 1.6))
     ax.set_xlabel('X')
     ax.set_ylabel('y')
-    ax.set_xlim(0, 2)
-    fignum += 1
+    
+    fig.tight_layout()
```

Comment:
Notice here I use `fig.tight_layout()` to avoid labels being cropped. Is this method preferred or would `subplots_adjust` be better?